frontend/app/routes/$lang+/_protected+/alerts+/index.tsx

```diff
@@ -17,9 +17,9 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   const userInfoToken: UserinfoToken = session.get('userInfoToken');
   const alertSubscription = await getSubscriptionService().getSubscription(userInfoToken.sin ?? '');
 
-  if (!alertSubscription) {
+  if (!alertSubscription || (alertSubscription.registered === false && alertSubscription.subscribed === false)) {
```

Comment:
I feel like this condition should be changed to only check if `alertSubscription.registered === false` (and leave out the `alertSubscription.subscribed === false` part).

Currently, if for whatever reason, `registered === false && subscribed === true`, this loader will redirect to `/subscribe/confirm`, but it should redirect to `/subscribe`.